installing/disconnected_install/installing-mirroring-disconnected-v2.adoc

```diff
@@ -0,0 +1,305 @@
+
+.Prerequisites
+
+* You have access to the internet to get the necessary container images.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the `oc-mirror` CLI plugin.
+* You have created the image set configuration file.
+
+.Procedure
+
+Copy the container images from the source specified in the image set configuration to the destination (container registry).
+
+[source,terminal]
+----
+$ oc mirror -c isc.yaml --workspace file://oc-mirror/mirror1 docker://localhost:6000 --v2
+----
+
+.Verification
+
+. Navigate into the `oc-mirror-workspace/` directory that was generated.
+. Navigate into the results directory, for example, `results-1639608409/`.
+. Verify that YAML files are present for the `ImageDigestMirrorSet`, `ImageTagMirrorSet` and `CatalogSource` resources.
+
+.Next steps
+
+* Configure your cluster to use the resources generated by oc-mirror.
+
+
+[id="oc-mirror-workflows-fully-disconnected"]
+=== Mirroring an image set in a fully disconnected environment
+
+In a fully disconnected environment, the workflow is composed of following 3 steps:
+
+* Mirror to disk: preparing an archive containing the image set that needs to be mirrored
+* A manual step, where the user transfers the archive to the network of the disconnected registry
+* Disk to mirror: from an environment that has access to the disconnected registry, execute oc-mirror to mirror the image set from the archive to the target disconnected registry.
+
+.Mirroring from mirror to disk
+
+You can use the oc-mirror plugin to generate an image set and save the contents to disk. The generated image set can then be transferred to the disconnected environment and mirrored to the target registry.
+
+Pulls the container images from the source specified in the image set configuration and pack them into a tar archive on disk (local directory).
+
+.Prerequisites
+
+* You have access to the internet to get the necessary container images.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the `oc-mirror` CLI plugin.
+* You have created the image set configuration file.
+
+.Procedure
+
+[source,terminal]
+----
+$ oc mirror -c isc.yaml 
+<file path> <1>
+--v2
+----
+<1> Add the desired file path 
+
+.Mirroring from disk to mirror
+
+Copies the container images from the tar archive to a container registry.
+
+[source,terminal]
+----
+$ oc mirror -c isc.yaml --from file://oc-mirror/mirror1 docker://localhost:6000 --v2
+----
+
+[id="oc-mirror-delete"]
+== Deleting the images from disconnected environment
+
+In v1, image pruning or deletion occurs automatically at the end of the mirroring process. To delete a specific release, catalog, or `additionalImage`, use the same `StorageConfig` settings. Update the `imagesetconfig` to exclude the content you wish to prune. The pruning logic will identify the differences and remove the images accordingly. Since this method has both advantages and disadvantages, it was modified in v2.
```

Comment:
Keeping v1 information in v2 docs is not making it confuse to the customers to understand?